scripts/roll_preload_list.py

```diff
@@ -118,11 +122,10 @@ def checkForDupes(parsedList):
 
 def main():
   args = getArgs()
-
   rawText = getRawText(args.preload_list_path)
-  pendingRemovals = []
+  pendingRemovals = getPendingAutomatedRemovals()
   if not args.skip_removals:
-    pendingRemovals = getPendingRemovals()
+    pendingRemovals += getPendingRemovals()
```

Comment:
I think if `skip_removals` is specified, all removals should be skipped.

Reply:
Done